.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -35,3 +35,4 @@ Steps to reproduce the behavior:
 ---
 **Contributor Resources**
 - See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
+- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
```

Comment:
```suggestion
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)

```

Reply:
Thanks for the suggestion will fix that.

Reply:
@thisiskaransgit updated with the suggestion